docs/glossary.asciidoc

```diff
@@ -1,11 +1,8 @@
-//[[ecs-glossary]]
-== Glossary of {ecs} Terms
+[[ecs-glossary]]
+=== Glossary of {ecs} Terms
 
-[[glossary-ecs]] 
+[[ecs-glossary-ecs]]
 ECS::
 Elastic Common Schema. A common set of document fields, field names, and their respective entity
 relationships to be used in the storage of log messages and other data in
```

Comment:
"log messages and other data"  should be "event data such as logs and metrics"

Reply:
Let's bring @karenzone's discussion about the definition of ECS here. We can use it in our glossary, and the top level glossary could then align with this one as well. So let's work on it here.

Karen, on March 18th:

> Elastic Common Schema (ECS)
>    A common set of document fields, field names, and their respective entity relationships to be used in the storage of log messages and other data in Elasticsearch.

My response, on the 19th:

> The Elastic Common Schema (ECS) defines a common set of fields, their datatype, and gives guidance on their correct usage. ECS is used to improve uniformity of event data ingested into Elasticsearch.

Reply:
incorporating your feedback Mike

> The Elastic Common Schema (ECS) defines a common set of fields, their datatype, and gives guidance on their correct usage. ECS is used to improve uniformity of event data ingested into Elasticsearch, such as logs and metrics.

Reply:
Here is an expanded definition that came up while editing the "Overview" section. Not sure if this might be helpful with the glossary as well:

The Elastic Common Schema (ECS) is an open source specification, developed with support from the Elastic user community, that defines a common set of fields to be used when storing event data, such as logs and metrics, in Elasticsearch. 

ECS specifies field names and Elasticsearch datatypes for each field, and provides descriptions and example usage. ECS also groups fields into ECS levels, and provides a set of naming guidelines for adding custom fields. 

The goal of ECS is to enable and encourage users of Elasticsearch to normalize their event data, so that they can better analyze, visualize, and correlate the data represented in their events.  ECS has been scoped to accommodate a wide variety of events, spanning;

- Event sources: whether the source of your event is an Elastic product, a third- party product, or a custom application built by your organization.
- Ingestion architectures: whether the ingestion path for your events includes Beats processors, Logstash, Elasticsearch ingest node, all of the above, or none of the above.
- Consumers: whether consumed by API, Kibana queries, dashboards, apps, or other means.

Reply:
Will update "What is ECS?" section with this

Reply:
But this is too long for the glossary definitions, though.

This discussion is also happening here https://github.com/elastic/stack-docs/pull/254 for the glossary of the main docs site